src/components/ModalPayment/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -27,13 +28,18 @@ const ModalConfirmation = (props: Props) => {
   const { purchaseType, amount } = useModalPaymentState(null);
   const dispatch = useModalPaymentDispatch(null);
   const modalRef = useScrollToElement();
+  const { t } = useTranslation();
 
   const closeModal = async (e: any) => {
     ReactPixel.trackCustom('ModalConfirmationButtonClick', {});
     e.preventDefault();
     // @TODO(wilsonj806) Replace the below with a proper fix
     //...for differentiating between a seller and a project
-    if (props.sellerId !== 'light-up-chinatown') {
+    if (
+      !props.projectId &&
```

Comment:
Why do we need to check `props.sellerId && props.sellerId !== 'light-up-chinatown'`? Isn't `!props.projectId` enough?

Reply:
I re-tested this and there actually wasn't anything passing for projectId. For now, checking for whether a seller exists or is equal to `light-up-chinatown` is enough